crates/uv/src/commands/auth/logout.rs

```diff
@@ -97,11 +97,14 @@ async fn pyx_logout(
     printer: Printer,
 ) -> Result<ExitStatus> {
     // Initialize the client.
-    let client = BaseClientBuilder::default()
-        .connectivity(network_settings.connectivity)
-        .native_tls(network_settings.native_tls)
-        .allow_insecure_host(network_settings.allow_insecure_host.clone())
-        .build();
+    let client = BaseClientBuilder::new(
+        network_settings.connectivity,
+        network_settings.native_tls,
+        network_settings.allow_insecure_host.clone(),
+        Preview::default(),
```

Comment:
We should be using the user-provided `Preview` value here

Reply:
added